Sources/WebDriver/Location.swift

```diff
@@ -0,0 +1,16 @@
+public struct Location: Codable {
+    public var latitude: Double
+    public var longitude: Double
+    public var altitude: Float
```

Comment:
This should be Double like the others, corresponding to the Javascript `number` type.